StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -175,13 +195,31 @@ class RowButton: UIView {
             }
         }
 
-        for view in [radioButton, imageView, labelsStackView].compactMap({ $0 }) {
+        for view in [radioButton, imageView, horizontalStackView].compactMap({ $0 }) {
             view.translatesAutoresizingMaskIntoConstraints = false
             view.isUserInteractionEnabled = false
             view.isAccessibilityElement = false
             addSubview(view)
         }
 
+        if let defaultBadge = defaultBadge {
+            for view in [labelsStackView, defaultBadge].compactMap({ $0 }) {
+                view.translatesAutoresizingMaskIntoConstraints = false
+                view.isUserInteractionEnabled = false
+                view.isAccessibilityElement = false
+                addSubview(view)
```

Comment:
Why are we adding `labelsStackView` and `defaultBadge` to the view again when it's already added to the `horizontalStackView` and `horizontalStackView` is added to the view? Also why not just iterate the arrangedSubviews in the `horizontalStackView`?